api/Clients.json

```diff
@@ -236,6 +228,56 @@
           }
         }
       },
+      "includeUncontrolled_option": {
+        "__compat": {
+          "description": "<code>includeUncontrolled</code> option",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```

Comment:
`matchAll()` landed in Chrome 47/Opera 32 and it included the `includeUncontrolled` option. Can you please fix throughout?

Reply:
Why not, hey!

Reply:
I apologize for not catching this yesterday. Thank you for doing this. I'm drinking from a fire hose when it comes to reviewing compat PRs. I really appreciate the help.

Reply:
BTW are most cases of ServiceWorker APIs consistent across webview, chrome, and android? I'm not so clued on the release model.

Reply:
There are minor differences here and there. I plan to run all of the sw data within the coming week.

Reply:
In general, push notifications aren't supported in WebView. 99% of the time Webview, Android, and Chrome have the same version number.